analytic_engine/src/sst/parquet/async_reader.rs

```diff
@@ -375,27 +367,23 @@ impl ObjectStoreReader {
             storage,
             path,
             meta_data,
-            metrics: ObjectReaderMetrics {
-                bytes_scanned: 0,
-                sst_get_range_length_histogram: metrics::SST_GET_RANGE_HISTOGRAM.local(),
-            },
+            begin: Instant::now(),
         }
     }
 }
 
 impl Drop for ObjectStoreReader {
     fn drop(&mut self) {
-        debug!("ObjectStoreReader is dropped, metrics:{:?}", self.metrics);
+        debug!(
+            "ObjectStoreReader dropped, path:{}, elapsed:{:?}",
+            &self.path,
+            self.begin.elapsed()
+        );
     }
 }
 
 impl AsyncFileReader for ObjectStoreReader {
     fn get_bytes(&mut self, range: Range<usize>) -> BoxFuture<'_, parquet::errors::Result<Bytes>> {
-        self.metrics.bytes_scanned += range.end - range.start;
```

Comment:
Those are covered in here
https://github.com/CeresDB/ceresdb/blob/9d8a56dd57a9b7177d8e076b22329f602f903682/components/object_store/src/metrics.rs#L188